rotkehlchen/tests/db/test_db_accounting_rules.py

```diff
@@ -123,3 +125,52 @@ def test_errors_with_rules(database: DBHandler) -> None:
             event_subtype=HistoryEventSubType.RECEIVE,
             counterparty='uniswap',
         )
+
+
+@pytest.mark.parametrize('db_settings', [{'include_crypto2crypto': True}])
+@pytest.mark.parametrize('counterparty', ['yabir'])
+def test_accountint_rules_overwrite(database: 'DBHandler', counterparty: str) -> None:
```

Comment:
Typo. Also don't forget to change the name here about the overwrite thing

```suggestion
def test_accounting_rules_overwrite(database: 'DBHandler', counterparty: str) -> None:
```